code/_onclick/hud/alert.dm

```diff
@@ -870,3 +870,56 @@ so as to remain in compliance with the most up-to-date laws."
 	if(!istype(usr))
 		return
 	living_owner.do_succumb(TRUE)
+
+/atom/movable/screen/alert/changeling_defib_revive
+	name = "Your heart is being defibrillated."
+	desc = "Click this status to be revived from fake death."
+	icon_state = "template"
+	timeout = 10 SECONDS
+	var/do_we_revive = FALSE
+	var/image/toggle_overlay
+
+/atom/movable/screen/alert/changeling_defib_revive/Initialize(mapload, parent_unit,)
+	. = ..()
+	var/image/defib_appearance = image(parent_unit)
+	defib_appearance.layer = FLOAT_LAYER
+	defib_appearance.plane = FLOAT_PLANE
+	overlays += defib_appearance
+
+/atom/movable/screen/alert/changeling_defib_revive/attach_owner(mob/new_owner)
+	. = ..()
+	RegisterSignal(owner, COMSIG_LIVING_DEFIBBED, PROC_REF(on_defib_revive))
```

Comment:
Do you know if you might need to remove this signal registration on detach from owner/destroy?

Reply:
Don't think its needed, but I'll add it.